addons/point_of_sale/models/pos_session.py

```diff
@@ -2082,29 +2077,29 @@ def get_total_sold_refund_per_category(self, group_by_user_id=None):
         else:
             return list(total_sold_per_user_per_category[0].items()), list(total_refund_per_user_per_category[0].items())
 
-    def load_order_pricelists(self, pricelist_ids):
-        params = {'search_params': {'domain': [('id', 'in', pricelist_ids)], 'fields': ['name', 'display_name', 'discount_policy']}}
-        pricelists = self.env['product.pricelist'].search_read(**params['search_params'])
+    def get_pos_ui_product_pricelists_by_ids(self, pricelist_ids):
+        pricelists = self.env['product.pricelist'].search_read(domain=[('id', 'in', pricelist_ids)], fields=['name', 'display_name', 'discount_policy'])
```

Comment:
my bad, use the loader_params and overwrite the domain so that we have the fields set in one place